### DISCUSSION TITLE
Discussion: retry helps the stability？

### DISCUSSION THREAD

**Discussion by HppZ** [microsoft/microsoft-ui-xaml#7437] [General]
![image](https://user-images.githubusercontent.com/12285582/139435603-049748d4-8b1b-42ce-8dc1-981e30da5516.png)

**Comment by StephenLPeters** [comment]
maybe something like this would help you? https://github.com/microsoft/microsoft-ui-xaml/blob/94c2d04f3bb2ce7609883aa8bedbd6cc07a1ff7c/dev/TeachingTip/InteractionTests/TeachingTipTests.cs#L1488
